pages/common/gpg2.md

```diff
@@ -0,0 +1,32 @@
+# gpg
+
+> GNU Privacy Guard 2.
+> Homepage: <https://docs.releng.linuxfoundation.org/en/latest/gpg.html>.
+
+- List imported keys:
+
+`gpg2 --list-keys`
+
+- Encrypt a specified file for a specified recipient, writing the output to a new file with `.gpg` appended:
+
+`gpg2 --encrypt --recipient {{alice@example.com}} {{path/to/doc.txt}}`
+
+- Encrypt a specified file with only a passphrase, writing the output to a new file with `.gpg` appended:
+
+`gpg2 --symmetric {{path/to/doc.txt}}`
+
+- Decrypt a specified file, writing the result to the standard output:
+
+`gpg2 --decrypt {{path/to/doc.txt.gpg}}`
+
+- Import a public key:
+
+`gpg2 --import {{path/to/public-key.gpg}}`
+
+- Export the public key of alice@example.com to the standard output:
```

Comment:
Here you mention "alice@example.com", but below you write "specified email address". I think we should use just one convention throughout the page. I would go with "specified email address".

Reply:
Oh yes, I think this was one of the propositions @sbrl suggested that I forgot.

I'll fix it.